repos/actors/checkuptime/libraries/checkuptime.py

```diff
@@ -0,0 +1,27 @@
+import os
+import time
+
+from leapp import reporting
+from leapp.libraries.stdlib import api
+
+
+def check_excessive_uptime():
+    # TODO: can this be parametrized?
```

Comment:
how the parametrization helps here? currently there is not a configuration for actors, and if user can parametrize it, how this is helpful?

Reply:
I think it could be beneficial for the customers to have the excessive uptime parameterized when building the repository RPM via some Makefile parameterization (e.g., env var with default value of 30 days).

Is this something that would be beneficial to customers? @swapdisk Or is hardcoded value of 30 days enough for this use case.
Anyway this would be done in OAMG-8807.

Reply:
Makefile parameterization (e.g., env var with default value of 30 days) would be nice. That way, we define a model for how parameterization can be done for future actors that may also benefit customers wanting to customize things. Makefile variables could be defined to customize which actors would be included in the package build.